### PR TITLE
Collaboration over Competition

### PR DESCRIPTION
**Pull request by itrich**

### CODE REVIEW

File: OpenInfraSummit2022/CollaborationOverCompetition.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# Collaboration over Competition - Fostering a community of CSPs
+
+## Abstract (1000 chars)
+
+The Sovereign Cloud Stack (SCS) is built upon the shoulders of giants: With proven technology such
+as OpenStack and Kubernetes the SCS project is building a modular software stack. SCS is much more
+than just another cloud stack - much rather SCS defines standards as part of a community process in
```
Comment (itrich) [inline OpenInfraSummit2022/CollaborationOverCompetition.md]: much rather SCS gathers an open community to join forces and define common standards.

File: OpenInfraSummit2022/CollaborationOverCompetition.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+which CSPs work together on their use-cases while also providing a reference implementation.
+Following the foundation the SCS builds upon the four opens are part of the core dna of the project - 
+however this is extended by a fifth pillar: Open Operations. The CSPs that engage in the community
+bring in their experiences, expertise and exchange not only solutions but also problems and hurdles
```
Comment (itrich) [inline OpenInfraSummit2022/CollaborationOverCompetition.md]: share their experience and learnings on the one hand and exchange on problems and hurdles that are faced in day to day operations of cloud platforms on the other hand.

File: OpenInfraSummit2022/CollaborationOverCompetition.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
+# Collaboration over Competition - Fostering a community of CSPs
```
Comment (itrich) [inline OpenInfraSummit2022/CollaborationOverCompetition.md]: Fostering a community of open operations

File: OpenInfraSummit2022/CollaborationOverCompetition.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+than just another cloud stack - much rather SCS defines standards as part of a community process in
+which CSPs work together on their use-cases while also providing a reference implementation.
+Following the foundation the SCS builds upon the four opens are part of the core dna of the project - 
+however this is extended by a fifth pillar: Open Operations. The CSPs that engage in the community
+bring in their experiences, expertise and exchange not only solutions but also problems and hurdles
+that are faced in day to day operations of cloud platforms.
+
+## Social Summary (100 chars)
+
+Collaboration over Competition - together we're not only stronger but it is also more fun ;)
```
Comment (itrich) [inline OpenInfraSummit2022/CollaborationOverCompetition.md]: Maybe: Collaboration over Competition - the cake is not a lie and it's big enough!

I really liked the pie symbol you presented during the SCS Community Summit (or maybe I'm just hungry...)

**Comment by fkr** [comment]
thanks for the review! Very good suggestions.

**Pull request merged**